File: Practical work 1, made in Java/app/src/main/java/com/example/practicalwork1/MainActivity.java
package com.example.practicalwork1;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        TextView textView = findViewById(R.id.mainText);
        textView.setText("This is practical work from Group 3\n\n");
        textView.append(getString(R.string.app_name));
        textView.append("\nThis application was developed by: ");
        textView.append(getString(R.string.dev_name));
    }
}

File: app/src/main/java/com/example/pracwork1/MainActivity.kt
package com.example.pracwork1

import android.os.Bundle
import androidx.activity.ComponentActivity
import androidx.activity.compose.setContent
import androidx.compose.foundation.layout.fillMaxSize
import androidx.compose.material3.MaterialTheme
import androidx.compose.material3.Surface
import androidx.compose.material3.Text
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier
import androidx.compose.ui.tooling.preview.Preview
import com.example.pracwork1.ui.theme.Pracwork1Theme

class MainActivity : ComponentActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContent {
            Pracwork1Theme {
                // A surface container using the 'background' color from the theme
                Surface(modifier = Modifier.fillMaxSize(), color = MaterialTheme.colorScheme.background) {
                    Greeting("Kristians Zondaks")
                }
            }
        }
    }
}

@Composable
fun Greeting(name: String, modifier: Modifier = Modifier) {
    Text(
            text = "This is the first application from Group 3:\n" +
                    "Kristians Zondaks\n" +
                    "Marks Cimmermanis\n" +
                    "Ieva Madara Glavecka\n" +
                    "This application was developed by $name",
            modifier = modifier
    )
}

@Preview(showBackground = true)
@Composable
fun GreetingPreview() {
    Pracwork1Theme {
        Greeting("Kristians Zondaks")
    }
}